examples/server-side-rendering/src/app/app.component.ts

```diff
@@ -73,17 +80,16 @@ export class AppComponent {
       ? this.injector.get('request')
       : undefined;
 
-    const searchParameters = parseServerRequest(req);
+    this.searchParameters = parseServerRequest(req);
```

Comment:
worth testing out on local if this still works as expected. To note that the approach in v4 is different

Reply:
that's done on my side, but feel free to test again.

Reply:
I've moved searchParameters back to the instance here 2b89b5f